modules/sql/db.go

```diff
@@ -0,0 +1,200 @@
+package sql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"sync"
+
+	// TODO: we can add more drivers from this list:
+	//  https://github.com/xo/dburl?tab=readme-ov-file#database-schemes-aliases-and-drivers
+	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/lib/pq"
+	_ "github.com/microsoft/go-mssqldb"
+	"github.com/xo/dburl"
+
+	"github.com/risor-io/risor/internal/arg"
+	"github.com/risor-io/risor/object"
+	"github.com/risor-io/risor/op"
+)
+
+const DB_CONN object.Type = "db_conn"
+
+type DB struct {
+	conn   *sql.DB
+	once   sync.Once
+	closed chan bool
+}
+
+func (db *DB) Type() object.Type {
+	return DB_CONN
+}
+
+func (db *DB) Inspect() string {
+	return "sql_conn()"
```

Comment:
To match conventions with types defined in other packages it looks like this should be `sql.conn`. For example in exec the Command type string is `exec.command`.

Reply:
updated, thanks. I was following the `pgx` module BTW.

Reply:
Roger, sounds like I need to update it to match.